sdk-extensions/autoconfigure/README.md

```diff
@@ -62,6 +62,12 @@ The [OpenTelemetry Protocol (OTLP)](https://github.com/open-telemetry/openteleme
+| otel.exporter.otlp.traces.client.key          | OTEL_EXPORTER_OTLP_TRACES_CLIENT_KEY          | The path to the file containing private client key to use when verifying an OTLP trace client's TLS credentials. The file should contain one private key PKCS8 PEM format. By default no client key file is used.                                                                                                                                                                                                    |
+| otel.exporter.otlp.metrics.client.key         | OTEL_EXPORTER_OTLP_METRICS_CLIENT_KEY         | The path to the file containing private client key to use when verifying an OTLP metric client's TLS credentials. The file should contain one private key PKCS8 PEM format. By default no client key file is used.                                                                                                                                                                                                   |
+| otel.exporter.otlp.client.certificate         | OTEL_EXPORTER_OTLP_CLIENT_CERTIFICATE         | The path to the file containing trusted certificates to use when verifying an OTLP trace, metric, or log client's TLS credentials. The file should contain one or more X.509 certificates in PEM format. By default no chain file is used.                                                                                                                                                                           |
+| otel.exporter.otlp.traces.client.certitifate  | OTEL_EXPORTER_OTLP_TRACES_CLIENT_CERTIFICATE  | The path to the file containing trusted certificates to use when verifying an OTLP trace server's TLS credentials. The file should contain one or more X.509 certificates in PEM format. By default no chain file is used.                                                                                                                                                                                           |
+| otel.exporter.otlp.metrics.client.certificate | OTEL_EXPORTER_OTLP_METRICS_CLIENT_CERTIFICATE | The path to the file containing trusted certificates to use when verifying an OTLP metric server's TLS credentials. The file should contain one or more X.509 certificates in PEM format. By default no chain file is used.                                                                                                                                                                                          |
```

Comment:
Can you document the log version of these as well for consistency? I know the spec doesn't mention the log version yet but we've already gone and added support / documentation for all the parameters so might as well be consistent.

Reply:
Done.
Can you please run the workflows? I would like to see if codecov will be OK this time or something else needs to be improved as well.